packages/search/src/content/index.ts

```diff
@@ -88,9 +125,17 @@ export class ContentSearchService
  * const searchResults = searchContent({ filters, options })
  * ```
  */
-export function searchContent(
+export async function searchContent(
   request: IContentSearchRequest = { filter: {}, options: {} }
 ): Promise<IContentSearchResponse> {
+  const siteCatalog = await getSiteCatalogFromOptions(request.options);
+  if (siteCatalog) {
+    request.filter = {
+      ...request.filter,
+      ...catalogToContentFilter(siteCatalog),
+    };
```

Comment:
This will overwrite any `group` array that was passed in on the `request.filter`. I'm thinking that's the right behavior to avoid unpredictable scoping. Open for feedback, though.

Reply:
Only scenario I can think of would be showing the content in a specific team, connected to a site... in our app we could just construct a IContentFilter that's specific to the team, or use `team.searchContent(...)` which will be part of the coarse-grained api. Long way of saying I think this is a good way to start

Reply:
This should be the opposite:

```ts
    request.filter = {
      ...catalogToContentFilter(siteCatalog),
      ...request.filter,
    };
```

I'm pretty sure we always prefer explicitly passed in params over any any params that come from context, in this case the site.

In our glorious future, this fn will be exectued from an instance of a site that will provide all the context, but also allow users to pass in overrides to that context.